src/routes/products.routes.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response, Router } from 'express'
+import getProducts from '../utils/queries/getProducts'
+
+/**
+ * Endpoint to fetch products based on vendor IDs.
+ *
+ *
+ * @route GET /products
+ * @param {Request} _req - The Express request object (unused).
+ * @param {Response} res - The Express response object.
+ * @returns {void}
+ *
+ * @throws {Error} If there is an error during database query.
+ *
+ * @example
+ * // GET /products
+ * // Returns an array of products from vendor 1.
+ */
+
+export default function (router: Router, db: any) {
+  router.get('/products', async (_req: Request, res: Response) => {
```

Comment:
we should do a more structured route. `/vendors/:vendor_id/products` so we aren't hard coding `1` in the query. It'll test them more on API usage.